### PR TITLE
WIP -- check tests and test locally with isotovideo on vars.json -- Extract log functions from bmwqemu

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: autotest.pm [hunk 1]
```diff
@@ -56,7 +57,7 @@ sub loadtest {
     eval $code;    ## no critic
     if ($@) {
         my $msg = "error on $script: $@";
-        bmwqemu::diag($msg);
+        log::diag($msg);
```
Comment (okurz) [inline autotest.pm]: "function of its own"? You mean package/file?

Comment (coolo) [inline autotest.pm]: right.

Comment (foursixnine) [inline autotest.pm]: Actually, the longer plan with #691 is to move it to a single package, and then use that one as base for bmwqemu... or simply add the logger everywhere (As bmwqemu is already present, i think inheritance can be a good trick too)

Ending up with:

```perl
package OpenQA::Log
use Log::Log4perl

package bmwqemu
use base OpenQA::Log
```

Comment (okurz) [inline autotest.pm]: yes, that sounds good

**Comment by okurz** [comment]
#691 should be better

**Pull request closed without merging**